pyrene/src/utils/DateUtils.ts

```diff
@@ -0,0 +1,264 @@
+import {
+  isExists, sub, add, Duration, format,
+} from 'date-fns';
+
+import {
+  zonedTimeToUtc, utcToZonedTime,
+} from 'date-fns-tz';
+
+const DAY = 'day';
+const MONTH = 'month';
+const YEAR = 'year';
+
+export const DATE_UNITS = {
```

Comment:
This is a use for a [ENUM](https://www.typescriptlang.org/docs/handbook/enums.html) :

```
export enum DATE_UNITS  {
  DAY,
  MONTH,
  YEAR,
};

```

Reply:
✅